### PR TITLE
improve logging

### PR DESCRIPTION
**Pull request by k9ert**
* We now have a `--tracerpc` switch which will dump each and every call to core/elements in the debug-logs like e.g.:
```
[  DEBUG] in             rpc: call(wallet/specter/mytrezor_2   ) payload:[{'method': 'listtransactions', 'params': ['*', 1000, 0, True], 'jsonrpc': '2.0', 'id': 0}]
[  DEBUG] in             rpc: call(wallet/specter/mytrezor_2   ) payload:[]
[  DEBUG] in             rpc: call(wallet/specter/mytrezor_2   ) payload:[{'method': 'getwalletinfo', 'params': [], 'jsonrpc': '2.0', 'id': 0}]
```
* We have a different logging-scheme for dev than for production (no timestamps and a bit more structure for dev)

### CODE REVIEW

File: src/cryptoadvance/specter/specter.py [hunk 1]
```diff
@@ -138,7 +138,8 @@ def cleanup_on_exit(self, signum=0, frame=0):
 
         logger.info("Closing Specter after cleanup")
         # For some reason we need to explicitely exit here. Otherwise it will hang
-        exit(0)
+        # Doesn't seem to be the case anymore?!
```
Comment (stepansnigirev) [inline src/cryptoadvance/specter/specter.py:141]: Still hangs forever for me

**Comment by stepansnigirev** [comment]
Looks good to me, except exit() thing above

File: src/cryptoadvance/specter/specter.py (hunk 1 above)
Comment (ben-kaufman) [reply src/cryptoadvance/specter/specter.py:141]: For me as well

Comment (k9ert) [reply src/cryptoadvance/specter/specter.py]: ok, recreated it

**Review by stepansnigirev — APPROVED** [review]

**Pull request merged**